site/pages/create-consequence-services/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -508,7 +508,7 @@ const CreateConsequenceServices = (props: CreateConsequenceServicesProps): React
                                     props.disruptionSummary ? stateUpdater(props.disruptionSummary, "description") : "";
```

Comment:
Can we change the name of this prop to disruptionDescription to reflect it is copy the description not a summary

Reply:
done